packages/frontend/src/components/ProjectAccess/ProjectAccess.tsx

```diff
@@ -140,8 +140,7 @@ const ProjectAccess: FC<ProjectAccessProps> = ({
                         {
                             type: 'organization',
                             role: convertOrganizationRoleToProjectRole(
-                                orgRoles[orgUser.userUuid] ||
-                                    OrganizationMemberRole.MEMBER,
+                                orgRoles[orgUser.userUuid],
```

Comment:
there's no need for the default "MEMBER" in 143 because if this is reaches and `orgRoles` & `organizationUsers` exist then something is really wrong.

Reply:
since `orgRoles` depends on `organizationUsers` this is safe and we don't need to check further